services/source-debugger/eventUploader.go

```diff
@@ -137,17 +126,20 @@ func uploadEvents(eventBuffer []*GatewayEventBatchT) {
 
 			//updating originalTimestamp in the event using the formula
 			//receivedAt - (sentAt - originalTimeStamp)
-			orgTS, err := time.Parse(time.RFC3339, ev.OriginalTimestamp)
+			originalTimestamp := ev["originalTimestamp"].(string)
```

Comment:
check if event has originalTimestamp property. Only events sent with SDK's have originalTimestamp. And maybe older SDK's do not have it also.
Also handle casting to string error?

Reply:
done. reusing the same logic from processor.go